discourse_rock/rockcraft.yaml

```diff
@@ -151,84 +143,70 @@ parts:
       grep -e ^gem plugin.rb >> Gemfile
     organize:
       "*": srv/discourse/app/plugins/discourse-prometheus/
-  get-data-explorer:
+  discourse-data-explorer:
     plugin: dump
-    after:
-      - get-discourse
+    after: [discourse, bundler-config]
     source: https://github.com/discourse/discourse-data-explorer.git
     source-commit: e7c19ac107dcd37618c7ac7b98530e99c7fe31db
     source-depth: 1
     organize:
       "*": srv/discourse/app/plugins/discourse-data-explorer/
-  get-patches:
+  patches:
     plugin: dump
+    after: [discourse]
     source: patches
     organize:
       "*": srv/discourse/app/patches/
   apply-patches:
     plugin: nil
     build-packages:
       - git
-    after:
-      - get-discourse
-      - get-patches
+    after: [discourse, patches]
     override-stage: |
+      craftctl default
```

Comment:
Thinking about it I don't think we need a `craftctl default` here, since the `plugin` is _nil_, so there is like...no default? So I removed it, see how this behaves now.